proposals/compatibility-mode.md

```diff
@@ -152,18 +172,25 @@ If either are used in the code passed to `createShaderModule` a validation error
 
 **Justification**: OpenGL ES 3.1 does not support `linear` interpolation nor `sample` sampling.
 
+Feature: `linear-interpolation-type`. Supported via the `noperspective` interpolation qualifier in desktop GLSL.
+Feature: `sample-interpolation-mode`. supported via `GL_OES_shader_multisample_interpolation` or OpenGL ES 3.2.
```

Comment:
nit: perhaps `interpolation-type-linear` and `interpolation-mode-sample` so they have a common prefix

Reply:
Done.